### PR TITLE
Feat/region model

### PR DESCRIPTION
**Pull request by hfzfdlh**
- add more functionality in model/region.py
- create region_controller.py
- create region_service.py

### CODE REVIEW

File: app/main/model/region.py [hunk 1]
```diff
@@ -1,32 +1,85 @@
...
+    city = db.Column(db.String(100), unique=True)
     created_at = db.Column(db.DateTime, nullable=False)
     updated_at = db.Column(db.DateTime, nullable=False)
 
-    def __init__(self, public_id, name, created_at, updated_at):
-        self.public_id = public_id
-        self.name = name
-        self.created_at = created_at
-        self.updated_at = updated_at
+    # def __init__(self, public_id, city, created_at, updated_at):
```
Comment (fatjan) [inline app/main/model/region.py:16]: should this commented lines be removed?

File: app/main/model/region.py [hunk 2]
```diff
@@ -1,32 +1,85 @@
...
+    #     self.public_id = public_id
+    #     self.city = city
+    #     self.created_at = created_at
+    #     self.updated_at = updated_at
 
     def __repr__(self):
-        return f"<Region(name={self.name})>"
+        return f"<Region(name={self.city})>"
```
Comment (fatjan) [inline app/main/model/region.py:23]: I think this should be:
```return f"<Region(city ={self.city})>"```

**Review by fatjan — CHANGES_REQUESTED** [review]
well done on the PR, just had some small comments. Once you've resolved them, kindly let me know.

File: app/main/model/region.py (hunk 1 above)
Comment (hfzfdlh) [reply app/main/model/region.py:16]: i was thinking that there will be a use for those lines of code, but i will delete it for now

File: app/main/model/region.py (hunk 2 above)
Comment (hfzfdlh) [reply app/main/model/region.py:23]: ok, correction on the way

**Review by fatjan — APPROVED** [review]

**Pull request merged**